labman/db/support_files/db_patch_manual.sql

```diff
@@ -2,6 +2,14 @@
 CREATE INDEX composition_notes_idx ON qiita.composition USING gin(to_tsvector('english', notes));
 CREATE INDEX plate_notes_idx ON qiita.composition USING gin(to_tsvector('english', notes));
 
+-- Add a column on the qiita_user table to enable access for labmanager
+ALTER TABLE qiita.qiita_user ADD COLUMN labmanager_access bool DEFAULT 'FALSE';
```

Comment:
Why not just create a table that has an FK relationship to `qiita.qiita_user`? There are easy possibilities for unintended consequences here, for instance, any method which previously performed `select * from qiita.qiita_user` is now getting a very different structure back.

cc @antgonza